src/components/Form.js

```diff
@@ -135,8 +135,35 @@ function Form(props) {
 
             // Validate the input for html tags. It should supercede any other error
             _.each(trimmedStringValues, (inputValue, inputID) => {
+
+                const workspaceRegex = inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+                const matches = inputValue.match(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+
+                let isMatch = CONST.WHITELISTED_TAGS.some(r => r.test(inputValue));
+                const leadingSpaceRegex = inputValue.search(CONST.VALIDATE_FOR_LEADINGSPACES_HTML_TAG_REGEX);
+
                 // Return early if there is no value OR the value is not a string OR there are no HTML characters
-                if (!inputValue || !_.isString(inputValue) || inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX) === -1) {
+                if (!inputValue || !_.isString(inputValue) || (leadingSpaceRegex === -1 && workspaceRegex === -1) ) {
+                    return;
+                }
+
+                // check for any matches that the original regex (workspaceRegex) matched
+                if(matches)
+                {
+                    // original regex does not catch leading spaces before tagnames 
```

Comment:
This comment doesn't seem helpful, no? Because we do check for leading spaces before tagnames now. We can update the comment on line 153-154 to say 

```
We need to check if all matched inputs are in the WHITELISTED_TAGS list. If they are, we can consider the input is valid and return early.
```

Reply:
Yes, basically return if there is any matched input that does not match in the WHITELISTED_TAGS.